t/api/03-auth.t

```diff
@@ -70,6 +77,7 @@ subtest 'access limiting for non authenticated users' => sub() {
         },
         'error returned as JSON'
     );
+    ok($mock_asset_remove_callcount == 0, 'asset deletion function was not called');
```

Comment:
I would use `is` for these checks. It gives a better error message when it fails.

Reply:
Done. nerdy note: this is arguably not *entirely* correct because `is` uses `eq`, not `==`. But it shouldn't matter in this case. Test::More does provide `cmp_ok` for using `==` when it matters, but meh, I think we're OK without it.